### PR TITLE
Feature/page refactor

### PR DESCRIPTION
**Pull request by TheSpartan1980**
# Description

Created Scenarios for:

- Create new form, go back to previous page
- Open an existing form, go back to previous page
- Edit the page path
- Create a section from Edit page

## Type of change

Please delete options that are not relevant.

- [x] Automated test

# How Has This Been Tested?

The test was created using the red-green-refactor cycle, the test has also been run in Chrome and Chrome headless locally and against Heroku.

# Checklist:

- [x] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and versioning
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto master and there are no code conflicts
- [ ] I have checked deployments are working in all environments
- [ ] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

File: smoke-tests/designer/features/startPage.feature [hunk 1]
```diff
@@ -12,4 +12,16 @@ Feature: Start Page
...
+
+  Scenario: Create new form, go back to previous page
+    Given I am on the form designer start page
+    And I chosen to create a new form
+    When I go back to previous page
+    Then the start page is displayed
+
+  Scenario: Open an existing form, go back to previous page
+    Given I am on the form designer start page
+    And I chosen to open an existing form
```
Comment (jenbutongit) [inline smoke-tests/designer/features/startPage.feature:25]: being a pedant.. "I chosen"? did you mean I've chosen or I choose?

File: smoke-tests/designer/features/steps/steps.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 const { Given, When, Then } = require("cucumber");
 const { formDesigner, previewPage } = require("../pageobjects/pages");
-const createComponent = require("../pageobjects/sections/createComponent.section");
-const AddLinkSection = require("../pageobjects/sections/addLink.section");
-const EditListSection = require("../pageobjects/sections/editLists.section");
-const EditPageSection = require("../pageobjects/sections/editPage.section");
-const EditSection = require("../pageobjects/sections/editSection.section");
-const { navMenu } = require("../pageobjects/sections");
-const FieldData = require("../../data/componentFieldData");
+const {
```
Comment (jenbutongit) [inline smoke-tests/designer/features/steps/steps.js:4]: noice!

File: smoke-tests/designer/features/steps/steps.js [hunk 3]
```diff
@@ -115,55 +117,53 @@ Then(
...
 When("I enter the details for my page", () => {
   this.newPageName = "Personal Details";
-  EditPageSection.pageTitle.setValue(this.newPageName);
-  EditPageSection.saveBtn.click();
+  editPage.pageTitle.setValue(this.newPageName);
+  editPage.saveBtn.click();
 });
 
 When("I link the {string} to the {string}", (fromPage, toPage) => {
   console.log(fromPage, toPage);
```
Comment (jenbutongit) [inline smoke-tests/designer/features/steps/steps.js:125]: do you need this anymore? can it be removed?

**Review by jenbutongit — CHANGES_REQUESTED** [review]

File: smoke-tests/designer/features/steps/steps.js (hunk 3 above)
Comment (TheSpartan1980) [reply smoke-tests/designer/features/steps/steps.js]: @jenbutongit good spot removed

**Review by jenbutongit — APPROVED** [review]

**Pull request merged**